### PR TITLE
Fix for issue with recurring events in certain timezones

### PR DESCRIPTION
**Pull request by adhorrig**
Issue 893:

When creating a recurring event using the dates:
Every Sunday from the 14th February -> 6th March.
The dates would change once you save the draft and go back to edit it, the dates would no longer display as the 14th February -> 6th March. They would be changed to the 15th February -> 7th March.

Video : https://drive.google.com/drive/folders/0B1eLynE0aU3mbVRmYmtlOHdtcjg

### CODE REVIEW

**Comment by tangentfairy** [comment]
@ckiss I know you had an opinion on how we did this before, maybe you could take a look and let me know what you think please? :)

**Comment by tangentfairy** [comment]
Related to https://github.com/CoderDojo/community-platform/issues/893

File: web/public/js/controllers/dojo-event-form-controller.js [hunk 1]
```diff
@@ -3,6 +3,7 @@
   'use strict';
 
   function getEveryTargetWeekdayInDateRange(startDateTime, endDateTime, targetWeekday, eventType) {
+    endDateTime = moment.utc(endDateTime).add(1, 'days');
```
Comment (tangentfairy) [inline web/public/js/controllers/dojo-event-form-controller.js]: Can you leave a comment explaining why we are adding a magic time of one day here?

**Comment by adhorrig** [comment]
When we do currentDate <= endDate, the while loop exists a day short

**Comment by tangentfairy** [comment]
:+1:

**Pull request merged**